core/frontend/src/store/mavlink.ts

```diff
@@ -64,7 +64,9 @@ class MavlinkStore extends VuexModule {
     if (message) {
       // TODO: Check if this is the best possible way to update `available_messages`
       // Reference: https://github.com/bluerobotics/blueos-docker/pull/508#discussion_r718729077
-      Vue.set(this.available_messages, message.messageName, message)
+      // We should not use `message.messageName` as dictionary key since it's a regex,
+      // the best approach is to use the message name as key
+      Vue.set(this.available_messages, (message.messageData.message as any).type, message)
```

Comment:
should we also update line 76? for consistency?

Reply:
Good catch!